crates/sui-protocol-config/src/lib.rs

```diff
@@ -490,6 +500,8 @@ impl ProtocolConfig {
                 stake_subsidy_rate: Some(1),
                 storage_gas_price: Some(1),
                 max_transactions_per_checkpoint: Some(1000),
+                // require 2f+1 + 0.75 * f stake for automatic protocol upgrades.
+                buffer_stake_for_protocol_upgrade_bps: Some(7500),
```

Comment:
Please move 7500 to protocol constants file and add a TODO to tune it

Reply:
This is the protocol constants file.

Reply:
Oh didn't know it got renamed

Reply:
yeah, because they aren't really constants anymore - the can change from one version to the next.